test/test_linalg.py

```diff
@@ -1277,9 +1329,28 @@ def gen_error_message(input_size, ord, keepdim, dim=None):
                 self.assertEqual(res_out.shape, expected.shape, msg=msg)
                 self.assertEqual(res_out.cpu(), expected, msg=msg)
 
+    # Test that linal.vector_norm gives the same result as numpy when inputs
+    # contain extreme values (inf, -inf, nan)
+    @unittest.skipIf(IS_WINDOWS, "Skipped on Windows!")
+    @unittest.skipIf(IS_MACOS, "Skipped on MacOS!")
+    @skipCUDAIfNoMagma
```

Comment:
Do we really need MAGMA or LAPACK? Seems like for vector norms it's never needed